app/models/cdo/manage/live.js

```diff
@@ -0,0 +1,48 @@
+const { errorPusherDefault } = require('../../../lib/error-helpers')
+
+/**
+ * @param {SummaryCdo[]} resultList
+ * @param {string} tab
+ * @param {{ column: string; order: 'ASC'|'DESC'}} sort
+ * @param backNav
+ * @param [errors]
+ * @constructor
+ */
+function ViewModel (resultList, tab, sort, backNav, errors) {
```

Comment:
We don't really need the errors or errorPusher since there are no possible validation errors in this model

Reply:
yes, that's true